pkg/k8shandler/clusterloggingrequest.go

```diff
@@ -24,15 +31,19 @@ func (clusterRequest *ClusterLoggingRequest) isManaged() bool {
 }
 
 func (clusterRequest *ClusterLoggingRequest) Create(object runtime.Object) error {
-	logrus.Debugf("Creating: %v", object)
+	logrus.Tracef("Creating: %v", object)
 	err := clusterRequest.client.Create(context.TODO(), object)
-	logrus.Debugf("Response: %v", err)
+	logrus.Tracef("Response: %v", err)
 	return err
 }
 
-func (clusterRequest *ClusterLoggingRequest) Update(object runtime.Object) error {
-	logrus.Debugf("Updating: %v", object)
-	return clusterRequest.client.Update(context.TODO(), object)
+//Update the runtime Object or return error
+func (clusterRequest *ClusterLoggingRequest) Update(object runtime.Object) (err error) {
+	logrus.Tracef("Updating: %v", object)
+	if err = clusterRequest.client.Update(context.TODO(), object); err != nil {
+		logrus.Errorf("Error updating %v: %v", object.GetObjectKind(), err)
+	}
+	return err
```

Comment:
I don't think this won't contain the value above... when written as you have it the scope is changed. that's why you can't do something like: 
```
if err, val := doTheThing(); err != nil {
  printTheError(err)
}

// val won't be initialized here
doSomethingWithTheVal(val)
```

Reply:
To what `val` are you referring in this context? `Update` takes an object and returns an error